src/java.desktop/share/classes/javax/swing/text/html/CSS.java

```diff
@@ -2300,6 +2310,19 @@ Object toStyleConstants(StyleConstants key, View v) {
             return family;
         }
 
+        @Override
+        public int hashCode() {
+            if (family != null) {
+                return family.hashCode();
+            } else {
+                return 0;
+            }
```

Comment:
```suggestion
            return (family != null) ? family.hashCode() : 0;
```
The conditional operator is perfect here, don't you think?